src/.pre-commit-config.yaml.jinja

```
@@ -91,6 +91,25 @@
   {%- set repo_rev.pyupgrade = "v2.38.2" %}
   {%- set repo_rev.ruff = "v0.6.8" %}
   {%- set repo_rev.setuptools_odoo = "3.1.8" %}
+{%- else %}
+  {%- set repo_rev.autoflake = "v2.3.1" %}
+  {%- set repo_rev.black = "25.1.0" %}
+  {%- set repo_rev.eslint = "9.35.0" %}
+  {%- set repo_rev.eslint_jsdoc = "57.0.8" %}
+  {%- set repo_rev.flake8 = "3.9.2" %}
+  {%- set repo_rev.flake8_bugbear = "21.9.2" %}
+  {%- set repo_rev.globals = "16.0.0" %}
+  {%- set repo_rev.isort = "6.0.1" %}
+  {%- set repo_rev.maintainer_tools = "f9b919b9868143135a9c9cb03021089cabba8223" %}
+  {%- set repo_rev.nodejs = "22.9.0" %}
+  {%- set repo_rev.odoo_pre_commit_hooks = "v0.1.6" %}
+  {%- set repo_rev.pre_commit_hooks = "v6.0.0" %}
+  {%- set repo_rev.prettier = "3.6.2" %}
+  {%- set repo_rev.prettier_xml = "3.4.2" %}
+  {%- set repo_rev.pylint_odoo = "v9.3.14" %}
```

Thank you @lmignon 

FYI https://github.com/OCA/pylint-odoo/pull/544

```suggestion
  {%- set repo_rev.pylint_odoo = "v9.3.15" %}
```